docs/advanced.md

```diff
@@ -337,3 +337,52 @@ Credential felix@krausefx.com:********* added to keychain.
 fastlane-credentials remove --username felix@krausefx.com
 password has been deleted.
 ```
+
+# Control configuration by lane and by platform
+
+In general, configuration files take only the first value given for a particular configuration item. That means that for an `Appfile` like the following:
+
+```ruby
+app_identifier "com.used.id"
```

Comment:
Until now, I think that's how it happens, however I don't think we ever documented or added tests for the order of those. Maybe by saying that's how it works, we should have at least one automated test for this behaviour (I don't think we have one currently)